rollup.config.js

```diff
@@ -46,6 +46,13 @@ const virtualImports = {
 const pagesDir = './src/lib/pages/';
 const pages = fs.readdirSync(pagesDir, 'utf-8').map((p) => join(pagesDir, p));
 
+// Suppress circular dependency warning
```

Comment:
Why do we need to suppress it?

Reply:
We don't need to, but it's a bit of an annoying message.